image/templates/helm/stackrox-central/templates/01-central-12-central-db.yaml

```diff
@@ -162,7 +162,6 @@ spec:
       - name: shared-memory
         emptyDir:
           medium: Memory
-          {{- /* Keep this in sync with shared_buffers in config/centraldb/postgresql.conf */}}
           sizeLimit: 2Gi
```

Comment:
@dashrews78 @erthalion do we even need this? From what I can tell, this was added due to a misunderstanding. Knowing this is not tied to `shared_buffers`, do we even still need this?

Reply:
That is a larger question that would require deeper research I suspect.  Since those that created this chart are not available to us, it is hard to get that context.  As such....https://issues.redhat.com/browse/ROX-29110

Reply:
It makes sense to have in some situations, e.g. when the container orchestrator limits `/dev/shm` to something small (it was happening on earlier K8s versions and Docker) and the database is running an analytical parallel query, which will allocate multiple large dynamic shared memory segments.